module/documents/actor/trait.mjs

```diff
@@ -269,8 +268,19 @@ export function getBaseItem(identifier, { indexOnly=false, fullItem=false }={})
   if ( !packObject ) return;
 
   // Build the extended index and return a promise for the data
-  const promise = packObject.getIndex({ fields: traitIndexFields() }).then(index => {
+  const fields = traitIndexFields();
+  const promise = packObject.getIndex({ fields }).then(index => {
     const store = index.reduce((obj, entry) => {
+      for (const field in fields) {
+        const val = foundry.utils.getProperty(entry, field);
+        if (field !== "system.type.value" && val !== undefined) {
+          foundry.utils.setProperty("system.type.value", val);
```

Comment:
Missing an argument here in setProperty.

Reply:
Should be fixed in my latest commit